tests/Test/AbstractIntegrationTestCase.php

```diff
@@ -359,9 +352,7 @@ private function getLinter(): LinterInterface
         static $linter = null;
 
         if (null === $linter) {
-            $linter = new CachingLinter(
-                getenv('FAST_LINT_TEST_CASES') ? new Linter() : new ProcessLinter()
-            );
+            $linter = new CachingLinter(new Linter()); // use the same linter as the "fix" command uses
```

Comment:
I think it's not related to the addition of a new linter. Please revert.
In tests, we use ProcessLinter. If we want to change that, let's open independent thread for it and not postpone this PR because of it ;)